### PR TITLE
improve SSH workflows

### PR DESCRIPTION
**Pull request by aminelouati**

### CODE REVIEW

**Review by marounkoussaifi — APPROVED** [review]

File: GetStarted/resources/catalog/SSH_Remote_Script_Execution.xml [hunk 1]
```diff
@@ -4,12 +4,12 @@
...
-    <variable name="SSH_CREDENTIAL_HANDLER" value="" model="PA:SPEL(variables[&#39;CREDENTIAL_TYPE&#39;] == &#39;SSH_PASSWORD&#39; ? showVar(&#39;SSH_PASSWORD&#39;) &amp;&amp; hideVar(&#39;SSH_PRIVATE_KEY&#39;) : showVar(&#39;SSH_PRIVATE_KEY&#39;) &amp;&amp; hideVar(&#39;SSH_PASSWORD&#39;))" description="SSH credential handler that shows and displays dynamically variables depending on the authentication type" group="SSH Connection" advanced="false" hidden="true"/>
+    <variable name="AUTHENTICATION_METHOD" value="SSH_PASSWORD" model="PA:LIST(SSH_PASSWORD,SSH_PRIVATE_KEY)" description="Authentication method to use either password or private key" group="SSH Connection" advanced="false" hidden="false"/>
+    <variable name="AUTHENTICATION_METHOD_HANDLER" value="" model="PA:SPEL(variables[&#39;AUTHENTICATION_METHOD&#39;] == &#39;SSH_PASSWORD&#39; ? showVar(&#39;SSH_PASSWORD&#39;) &amp;&amp; hideVar(&#39;SSH_PRIVATE_KEY&#39;) : showVar(&#39;SSH_PRIVATE_KEY&#39;) &amp;&amp; hideVar(&#39;SSH_PASSWORD&#39;))" description="Allows users to use either the SSH Password Authentication method or the Private Key Authentication method" group="SSH Connection" advanced="false" hidden="true"/>
```
Comment (fviale) [inline GetStarted/resources/catalog/SSH_Remote_Script_Execution.xml:12]: handler which enables SSH_PASSWORD field or SSH_PRIVATE_KEY based on the chosen AUTHENTICATION_METHOD

File: SSH/resources/catalog/SSH_Terminal_Via_Browser.xml [hunk 2]
```diff
@@ -4,12 +4,13 @@
...
+    <variable name="SSH_PASSWORD" value="ENC(dP8Lz2T+xXBd+XWKjUzijg==)" model="PA:HIDDEN" description="SSH server password" group="SSH Connection" advanced="false" hidden="false"/>
+    <variable name="SSH_PRIVATE_KEY" value="${SSH_USER}@${SSH_TARGET_HOST}" model="PA:CREDENTIAL" description="SSH private key" group="SSH Connection" advanced="false" hidden="true"/>
+    <variable name="AUTHENTICATION_METHOD" value="SSH_PASSWORD" model="PA:LIST(SSH_PASSWORD,SSH_PRIVATE_KEY)" description="Authentication method to use either password or private key" group="SSH Connection" advanced="false" hidden="false"/>
+    <variable name="AUTHENTICATION_METHOD_HANDLER" value="" model="PA:SPEL(variables[&#39;AUTHENTICATION_METHOD&#39;] == &#39;SSH_PASSWORD&#39; ? showVar(&#39;SSH_PASSWORD&#39;) &amp;&amp; hideVar(&#39;SSH_PRIVATE_KEY&#39;) : showVar(&#39;SSH_PRIVATE_KEY&#39;) &amp;&amp; hideVar(&#39;SSH_PASSWORD&#39;))" description="Allows users to use either the SSH Password Authentication method or the Private Key Authentication method" group="SSH Connection" advanced="false" hidden="true"/>
```
Comment (fviale) [inline SSH/resources/catalog/SSH_Terminal_Via_Browser.xml:13]: see description above

**Review by fviale — APPROVED** [review]
LGTM

**Review by nebilbenmabrouk — APPROVED** [review]
LGTM

**Review by mbenguig — APPROVED** [review]

**Pull request merged**